specification/source/glossary.rst

```diff
@@ -70,3 +85,13 @@ the Open Building Controls project.
    Mode
      In CDL, by mode we mean a signal that can take on multiple distinct
      values, such as ``On``, ``Off``, ``PreCool``.
+
+   Non-HVAC System
+     Any non-HVAC plant coupled with the control system.
+
+   Plant
```

Comment:
We should also use "plant" for "a dedicated functional units for controls" as ISO does. This is customary in the control literature where you always have the separation between a "controller" and a "plant".

Maybe we should not use "non-HVAC system" but rather be specific, as a non-HVAC system could be an electrical system, an active facade, etc.